src/pymortests/demos.py

```diff
@@ -34,6 +34,8 @@
     ('burgers', ['--num-flux=lax_friedrichs', '0.1']),
     ('burgers', ['--num-flux=engquist_osher', '0.1']),
     ('burgers', ['--num-flux=simplified_engquist_osher', '0.1']),
+    ('neural_networks', [100, 100, 20]),
+    ('neural_networks_fenics', [100, 20]),
```

Comment:
Can you say how the runtime compares with other demos?

Reply:
The first one is quite fast, on my machine about 3 seconds, similar to many other examples. However, the second one takes much longer due to the necessity of nonlinear solving. On my machine about 2 minutes. Maybe we should leave that one out?

Reply:
Even with a much smaller problem size the check would still be meaningful imo. If you can drastically reduce the runtime w/o instability or some other problem, I say keep it in. 

I was able to reproduce the CI failure in the demos test with a memory limit on the pytest container.  And the nonlinear one would be my first suspect for the memory usage. So maybe reducing the runtime via the problem size will also take care of the CI failure.

Reply:
The smaller example now takes few seconds on my machine, so that should work quite nice.